developers/sitl.md

```diff
@@ -53,3 +53,12 @@ To run ArduSub SITL with Gazebo, it's necessary to:
 The console will start to display the output if the connection was successful.
 
 <img src="/images/gazebo_sitl.gif" class="img-responsive img-center" style="max-height:400px;">
+
+## Troubleshooting
+
+1. Check if all dependencies are installed via [install some required packages](http://ardupilot.org/dev/docs/setting-up-sitl-on-linux.html#install-some-required-packages).
+    * If you are running an Unix OS, it's possible also to install missing dependencies with your package manager or `pip`.
+2. Be sure that mavproxy is installed with `whereis mavproxy`, `pip list | grep mavproxy -i` or just `mavproxy`.
+    * If mavproxy is not installed, please run the install packages script (described in step 1) or install the packages manually.
+3. Be sure to follow [add some directories to your search path](http://ardupilot.org/dev/docs/setting-up-sitl-on-linux.html#add-some-directories-to-your-search-path-facultative). It's possible to check with `echo $PATH | grep ardupilot`, it should display the `*/ardupilot/Tools/autotest` path.
```

Comment:
the solution for point 3 is for a missing installation, but doesn't tell you how to add to the search path? Should  this be a point 3 and point 4?

 Maybe we should just improve the installation instructions here, making a complete explicit instruction and all the necessary info can be on this page. Then this troubleshooting section should not be necessary (or at least all the current points would be solved above on this page).